components/empty-state/src/empty-state.ts

```diff
@@ -50,6 +50,8 @@ const ITEMS_HORIZONTAL_BREAKPOINT = 540;
 /** The min-width from which the empty state items are aligned next to each other. */
 const LAYOUT_HORIZONTAL_BREAKPOINT = 760;
 
+declare const window: any;
```

Comment:
NIT: you can declare your own typings till it is part of the `lib.dom` package: microsoft/TypeScript#28502


```typescript
/** Workaround until created: microsoft/TypeScript#28502 */
interface ResizeObserver {
...
}
```

```suggestion
declare const window: { ResizeObserver?: ResizeObserver };
```

Reply:
Nope, doesn't work. Tried to type it with 4 typescript specialists at my side... did not work. At some point of our nested tsconfig structure, either the types, files, typeroots or includes are being overwritten... 
this is sad 😢